style/annotator-content-analysis.scss

```diff
@@ -0,0 +1,23 @@
+.survey-unit {
+  position: relative;
+  // top: 0;
+  right: 0;
+
+  &__topic {
+    font-size: 1.2em;
+    &--name {
```

Comment:
this is BEM syntax. i'm trying to use it more often though I'm not really using it to its full extent
https://css-tricks.com/bem-101/